mrdo_arcade.asm

```diff
@@ -8996,11 +9036,11 @@ MR_DO_PUSH_RIGHT_02_PATW:
 ;	DB 000,000,192,224,144,176,224,200
 ;	DB 248,056,128,128,128,000,000,000
 
-FIVE_HUNDRED_SCORE_PAT:
-   DB 000,000,000,000,000,113,066,114
-   DB 010,074,049,000,000,000,000,000
-   DB 000,000,000,000,000,140,082,082
-   DB 082,082,140,000,000,000,000,000
+;FIVE_HUNDRED_SCORE_PAT:

```

Comment:
If you keep my BONUS tiles loaded in the tileset, you can remove this pattern from the SPT 
My new tiles for the 500 and 1000 signs are at 208 and following